src/app/homeseeker/display/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, Card } from "@tremor/react";
+import Link from "next/link";
+import { getSchedules } from "@/db/homeseeker/schedule";
+
+export default async function Home() {
+	const schedules = await getSchedules();
+
+	return (
+		<div className="container mx-auto py-20">
+			<h1 className="mb-5 text-tremor-metric font-medium">Schedules</h1>
+			<div className="grid grid-cols-2 gap-5">
+				{schedules.map((schedule) => (
+					<Card key={schedule.id}>
+						<div className="mb-3 flex items-center">
+							<div>
+								<h2 className="text-tremor-title font-medium">{schedule.id}</h2>
+								<p className="text-slate-600 text-sm">
+									Start time: {new Date(schedule.start).toLocaleString()}
+								</p>
+								<p className="text-slate-600 text-sm">
+									End time: {new Date(schedule.end).toLocaleString()}
+								</p>
+							</div>
+							<Link href={{
+								pathname: '/homeseeker/makeappointment', query: {
+									schedule: schedule.id
+								}
```

Comment:
Not sure if this is Biome's doing or not, but the formatting is a bit weird.
```suggestion
								pathname: '/homeseeker/makeappointment',
								query: { schedule: schedule.id },
```

Reply:
Probably i didn't turn off prettier when doing this project that's probably why.